content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
-  time="2024-02-19T10:16:53Z" level=info msg="bpf Start successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="controller Start successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="command StartServer successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="start write CNI config\n" subsys="cni installer"
-  time="2024-02-19T10:16:53Z" level=info msg="kmesh cni use chained\n" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="Copied /usr/bin/kmesh-cni to /opt/cni/bin." subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="kubeconfig either does not exist or is out of date, writing a new one" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="wrote kubeconfig file /etc/cni/net.d/kmesh-cni-kubeconfig" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="command Start cni successful" subsys=manager
-  ```
+After the applications have been manage by kmesh, we need to test that they are still working properly.
 
+```console
+k exec deploy/sleep -- curl -s 10.244.0.21 | grep -o "<title>.*</title>"
+<title>httpbin.org</title>
+```
 
+Note: 10.244.0.21 is the IP of httpbin
 
-## Compile and Build
+## Clean Up
 
-### Source code compilation
+If you don't want to use kmesh to govern the application anymore, you can delete the labels on the namespace and restart the pod.
 
-- Code download
+```console
+kubectl label namespace default istio.io/dataplane-mode-
+kubectl delete po httpbin-65975d4c6f-96kgw sleep-7656cf8794-8tp9n
+kubectl describe po httpbin-65975d4c6f-h2r99
 
-  ```sh
-  [root@ ~]# git clone https://github.com/kmesh-net/kmesh.git
-  ```
+Name:             httpbin-65975d4c6f-h2r99
+Namespace:        default
+Priority:         0
+Service Account:  httpbin
+Node:             kmesh-control-plane/172.18.0.2
+Start Time:       Fri, 26 Apr 2024 15:49:38 +0800
+Labels:           app=httpbin
+                  pod-template-hash=65975d4c6f
+                  version=v1
+Annotations:      <none>
+Status:           Running
+IP:               10.244.0.28
+```
 
-- Code compilation
+Delete kmesh:
 
-  ```sh
-  [root@dev tmp]# cd kmesh/
-  [root@dev Kmesh]# make build
-  ```
+```console
+kubectl delete daemonset -n kmesh-system kmesh
+kubectl delete ns kmesh-system
+```
 
-  Kmesh will be compiled and built within the build image, and the build artifacts will be output to the `out` directory.
+To remove the sleep and notsleep applications:
 
-  ```bash
-  [root@localhost kmesh]# ls out/amd64/
-  kmesh-cmd  kmesh-daemon       libbpf.so    libbpf.so.0.8.1       libkmesh_deserial.so  libprotobuf-c.so.1      mdacore
-  kmesh-cni  libboundscheck.so  libbpf.so.0  libkmesh_api_v2_c.so  libprotobuf-c.so      libprotobuf-c.so.1.0.0
-  ```
+```console
+kubectl delete svc httpbin sleep
+kubectl delete deploy httpbin sleep
+kubectl delete serviceaccount httpbin sleep
```

Comment:
If we put the configuration of sample applications in some places, we could `kubectl delete -f` them here directly